README.md

```diff
@@ -122,25 +138,53 @@ For details about targets spawned by this macro please refer to `Implementation`
 This file contains simple helper functions written in starlark as well as macro `build_openroad()`.
 The implementation of this macro spawns multiple `genrule` native rules which are responsible for preparing and running ORFS physical design flow targets during bazel build stage.
 
-There are 5 kinds of genrules spawned in this macro:
+There are 6 kinds of genrules spawned in this macro:
 
 * Config generation targets
   * Common for the whole design (named: `target_name + “_config”`)
   * ORFS stage-specific config (named: `target_name + “_” + stage + “_config”`)
 * Stage targets (named: `target_name + “_” + stage`)
+  * Special stage: Memory targets (named: `target_name + “_memory”`)
+  * Special mock flow: Mock Area targets (named: `target_name + “_” + stage + “_mock_area”`)
 * Make targets (named: `target_name + “_” + stage + “_make”`)
-* Mock Area targets (named: `target_name + “_” + stage + “_mock_area”`)
-* Memory targets (named: `target_name + “_memory”`)
 
 There are two kinds of flows available:
-* docker flow (Stage targets)
-* local flow (Make targets)
+* Docker flow (Stage targets)
+* Local flow (Make targets)
+
+Both docker and local flow does the same thing: for each stage of the physical design flow it writes config files, sets env vars pointing to those files, builds a command line to execute in ORFS environment and runs it through the `entrypoint` script.
+
+#### Docker flow
+
+Docker flow uses containerized environment with preinstalled ORFS to run the physical design flow.
+Example targets which run the docker flow include:
+
+* //:L1MetadataArray_test_floorplan
+* //:L1MetadataArray_test_memory
+* //:tag_array_64x184_synth
+
+It implicitly depends on a docker image with installed ORFS environment being present in docker runtime of the machine running bazel targets.
+The docker image used in the flow defaults to `ghcr.io/antmicro/openroad-flow-scripts/ubuntu22.04:latest`.
+It can be also overwritten per `build_openroad` instance with a `docker_image` attribute.
```

Comment:
"overwritten" is confusing here. "The default can be overridden.."

Reply:
fixed